solidity-v1/dashboard/src/components/modal/threshold/WithdrawGrantedTokens.jsx

```diff
@@ -0,0 +1,195 @@
+          </form>
+          <OnlyIf condition={grants.length > numberOfGrantsDisplayed}>
+            <div
+              className="withdraw-granted-tokens__view-more-container"
+              onClick={() =>
+                setNumberOfGrantsDisplayed(numberOfGrantsDisplayed + 5)
+              }
+            >
+              <span>+&nbsp;</span>
+              View More
+            </div>
+          </OnlyIf>
+        </div>
+      </ModalBody>
+      <ModalFooter>
+        <SubmitButton
+          className="btn btn-primary btn-lg mr-2"
+          type="submit"
+          onSubmitAction={(awaitingPromise) => {
+            onWithdrawClick(awaitingPromise)
+          }}
+          disabled={formik.values.grantIds.length === 0}
+        >
+          withdraw
+        </SubmitButton>
+        <Button className={`btn btn-unstyled text-link`} onClick={onClose}>
+          Cancel
+        </Button>
+      </ModalFooter>
+    </>
+  )
+})
+
+const renderGrant = (grant, totalNumberOfGrants, formik) => {
+  return (
+    <div
+      key={`Grant-${grant.id}`}
+      style={{
+        display: "flex",
+        alignItems: "baseline",
+      }}
+    >
+      <FormCheckboxBase
+        className="withdraw-granted-tokens__form_checkbox mr-1"
+        name="grantIds"
+        type="checkbox"
+        value={grant.id}
+        checked={formik.values.grantIds.includes(grant.id)}
+        onChange={(e) => {
+          const set = new Set(formik.values.grantIds)
+          if (e.target.checked) {
+            set.add(grant.id)
+          } else {
+            set.delete(grant.id)
+          }
+          formik.setFieldValue("grantIds", [...set])
+        }}
+      />
+      <Accordion
+        allowZeroExpanded={totalNumberOfGrants > 1}
+        className={"withdraw-granted-tokens__grants-accordion"}
+      >
+        <AccordionItem
+          {...(totalNumberOfGrants === 1
+            ? { dangerouslySetExpanded: true }
+            : {})}
+        >
+          <AccordionItemHeading>
+            <AccordionItemButton>
+              <div className="withdraw-granted-tokens__token-amount">
+                <TokenAmount
+                  amount={grant.readyToRelease}
+                  amountClassName={"h4 text-mint-100"}
+                  symbolClassName={"h4 text-mint-100"}
+                  token={KEEP}
+                />
+              </div>
+              <span className="withdraw-granted-tokens__details-text">
+                Details
+              </span>
+              <OnlyIf condition={totalNumberOfGrants > 1}>
+                <span className="withdraw-granted-tokens__expand-button" />
+              </OnlyIf>
+            </AccordionItemButton>
+          </AccordionItemHeading>
+          <AccordionItemPanel>
+            <div className="withdraw-granted-tokens__info-row">
+              <span className="withdraw-granted-tokens__info-row-title withdraw-granted-tokens__info-row-title--small">
```

Comment:
Same as above. I think we can get the same result by using existing classes.

Reply:
I think we can't - we have `text-transform`, `font-size` and `font-weight` set here.

Reply:
As I can see on Figma it's typography style so maybe we can create this css class instead of creating a special class name and seems that this class could be useful in other places.

![image](https://user-images.githubusercontent.com/57687279/142387180-a95bf36a-6dca-4451-a022-305fff1f9520.png)

Reply:
Regarding `withdraw-granted-tokens__info-row-value withdraw-granted-tokens__info-row-value--small`- we can use `text-small text-grey-60`.

Reply:
Good catch! Fixed in 72e48c86fb515af7ab87ef452405d88b8e363b58